docs/deployment/microbit-deployment-draft.md

```diff
@@ -0,0 +1,166 @@
+
+### URL versioning scheme
+
+For each published major version, the deployed editor will live at
+
+`python.microbit.org/v<major version>/editor.html`
+
+Because the use of semantic versioning ensures that things will not break for the user when upgrading minor versions, there will not be fine-grained control over which minor version a user gets. 
+
+For example visiting `python.microbit.org/v2/` will serve version `2.<latest minor>`
+
+### Versions in URLs
+
+When a script is published or shared, it must be published including the major version of the editor (and therefore the MicroPython stack) that it was created with so that anyone following the link will be taken back to the version the script was created with.
+
+If a user visits python.microbit.org/latest/ and is working on the editor there, the URL published should still be versioned, such as python.microbit.org/v1/. 
+
+See “workflows” for discussion of how the user will be notified about newer versions
+
+
+### Versioning in hex files
+
+When the user is provided a hex file, the full semantic editor version must also be embedded into the hex file at a known location so that when unpacking the hex, any version of the editor can offer to load the version that the script was created in.
+
+The hex file must also contain some editor unique identity, so that any hex can be related to the appropriate editor, perhaps automatically (e.g. MP signature at known location).This could be part of the version number?
+
+See “workflows” for discussion of how the user will be notified about the fact they are heading to an older version.
+
+(This kind of thing can enable the ability to show a user the exact version of the source code that was used to generate their hex file, including MicroPython code)
+
+## Release cadence
+
+Because we must commit to maintaining and fixing versions that are deployed on python.microbit.org it is desirable to make a small number of major release. 
+
+We propose an annual cadence for major releases, with an RC released early in the year so that a stable release can be made in July after the school year ends. This gives teachers time to become familiar with it before the school year start. The previous year’s version will remain available after this but will no longer be redirected to via python.microbit.org/latest or python.microbit.org/
+
+For the first stable release (Ideally made in July 2017) there should be only a 6-month lifetime to allow a greater number of changes while the project is growing.
+
+## Unstable Versions
+
+There should be a mechanism to deploy versions of the editor that are not yet considered stable. These will not be widely exposed to the majority of visitors to the site, but will be published for enthusiasts, developers and other interested parties to help test the editor.
+
+It should be very hard for a naive user to wind up using a beta by accident. (Think, about:config warning in Firefox, etc)
```

Comment:
I think those statements are compatible :) I think we need to allow ourselves to kill versions that were either 
 * never stable
 * released more than 3 years ago

Otherwise things might get a bit hectic!